task5/src/com/bsu/CheckServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.bsu;
```

Comment:
Mind that the standard project folder structure is violated here
It is **`src\main\java`**`\your\package\File.java`